Kernel/Devices/Storage/AHCI/Definitions.h

```diff
@@ -543,7 +542,7 @@ enum DeviceSignature : u32 {
 
 namespace Kernel {
 
-struct [[gnu::packed]] ATAIdentifyBlock {
+struct ATAIdentifyBlock {
```

Comment:
This actually seems to be the only struct needing a `[[gnu::packed]]` in this file.
You probably didn't notice that due to it missing an `AssertSize`. It might be nice to add one.